index.xml

```diff
@@ -232,6 +265,21 @@ MUST be ignored.
 
    <section anchor="canonicalization" title="Signature String Construction">
 
+   <t>
+A signed HTTP message needs to be tolerant of some trivial
+alterations during transmission
+as it goes through gateways, proxies, and other entities. These changes are often
+of little consequence and very benign, but also often not visible to or
+detectable by either the sender or the recipient. Simply signing the entire
+message that was transmitted by the sender is therefore not feasible:
+Even very minor changes would result in a signature which cannot be verified.
+   </t>
+   <t>
+This specification allows the sender to select which headers are meaningful
+by including their names the `headers` signature parameter. The headers
```

Comment:
missing "in". It should be "by including their names in the `headers` signature"

Reply:
Resolved 54dc50addae904b8a42cd152456bf53fdf438256